xcode/Subconscious/Shared/Services/AddressBookService.swift

```diff
@@ -400,7 +400,11 @@ actor AddressBookService {
         preventOverwrite: Bool = false
     ) -> AnyPublisher<Void, Error> {
         Future.detached {
-            try await self.followUser(did: did, petname: petname)
+            try await self.followUser(
+                did: did,
+                petname: petname,
+                preventOverwrite: preventOverwrite
```

Comment:
This parameter as not being passed before, which is why overwriting was even possible. It was a regression in the first place.